### DISCUSSION TITLE
I do not know the relationship between this repo and SAP.com

### DISCUSSION THREAD

**Discussion by stevending1st** [hypertrons/hypertrons-crx#182] [Q&A] [answered]
When I pulled request ( #180 ), I found I should sign the [CLA](https://cla-assistant.io/hypertrons/hypertrons-crx?pullRequest=180).
I see:

> Thank you for your interest in contributing to open source software projects (“Projects”) made available by **SAP** SE or its affiliates (“**SAP**”)...

But I do not know the relationship between this repo and SAP.

**Comment by LiuChangFreeman** [comment, marked as answer]
We have no relationship with SAP and it might be a mistake for the default template.   
Another version without SAP may be available within next days.

**Reply by stevending1st** [reply]
Thx
